cli/lib/cli.js

```diff
@@ -100,6 +100,7 @@ const descriptions = {
   browserOpenMode: 'path to a custom browser to be added to the list of available browsers in Cypress',
   browserRunMode: 'runs Cypress in the browser with the given name. if a filesystem path is supplied, Cypress will attempt to use the browser at that path.',
   cacheClear: 'delete all cached binaries',
+  cachePurge: 'removes all cached binaries but for the current version',
```

Comment:
More clarifying language that it keeps *your* version you are using - not the most current version of Cypress.

Use word 'delete' to be more clear.

```suggestion
  cachePurge: 'deletes all cached binaries except for the version currently in use',
```